### ISSUE TITLE
how to decrease high loss when training more than 100 classes

### ISSUE THREAD

**Issue by tomjerrygithub** [AlexeyAB/darknet#3411]
I am training a model with more than 100 classes. The loss value lasted for two or three days at about 6.

I used focal loss, the training data re-sampled, a total of 3000000 .

I set random = 1  lr init 0.01 

How to reduce the loss value? any suggestion ?   thx ~

**Comment by AlexeyAB** [comment]
Try to decrease learning rate and continue training.

What model do you use?

**Comment by tomjerrygithub** [comment]
> Try to decrease learning rate and continue training.
> 
> What model do you use?

yolov3   more than 100 classes objects detection ；
and decrease lr and continue training   i will have a try
